src/main/java/fr/litopia/model/Location.java

```diff
@@ -53,11 +55,28 @@ public Integer getTemps() {
         return temps;
     }
 
-    public void setTemps(Integer temps) {
-        this.temps = temps;
-    }
-
     public Long getId() {
         return id;
     }
+
+    public void cloreLocation(Bornette bornette) {
+        velo.setBornette(bornette);
+        temps=toIntExact(ChronoUnit.MINUTES.between(depart,LocalDateTime.now()));
+        prix=temps*(velo.getModele().getPrixHoraire()/60);
+    }
+
+    public Boolean isUnderFiveMinutes(){
+        if(toIntExact(ChronoUnit.MINUTES.between(depart,LocalDateTime.now()))<5){
+            return true;
+        }
+        else{
+            return false;
+        }
```

Comment:
```suggestion
        return toIntExact(ChronoUnit.MINUTES.between(depart,LocalDateTime.now()))<5
```
C'est la même chose, mais en plus simple